src/sentry/static/sentry/app/views/organizationGroupDetails/groupEventDetails/groupEventDetails.tsx

```diff
@@ -163,53 +166,77 @@ class GroupEventDetails extends React.Component<Props, State> {
       eventError,
       onRetry,
     } = this.props;
+
     const evt = withMeta(event);
 
+    // reprocessing
+    const hasReprocessingV2Feature = project.features?.includes('reprocessing-v2');
+    const {activity, statusDetails, status} = group;
+    const {pendingEvents} = statusDetails;
+
+    const {data, type} =
+      orderBy([...activity], ({dateCreated}) => new Date(dateCreated), ['desc'])[0] || {};
```

Comment:
Could a comment from a user while reprocessing is going cause the status to be incorrectly displayed?

Reply:
I would have to check that... @untitaker  please do you know about it?

Reply:
during reprocessing it should not be possible to create any activity at all on the issue.